File: JanperCab/5-JanperCab/ClientApp/src/app/_models/duraform-order/DuraformOrderForListDto.ts
import { OrderStatus } from './../../_enums/OrderStatus';
export class DuraformOrderForListDto {
  id: string;
  orderNumber: number;
  customerOrderNumber: string;
  orderStatus: OrderStatus;
  distributorId: number;
  cabinetMakerId: number;
  distributorName: string;
  cabinetMakerName: string;
  description: string;
  createdDate: Date;
}

File: JanperCab/3-Application/Dtos/DuraformOrder/DuraformOrderForListDto.cs
using _1_Domain;
using System;

namespace _3_Application.Dtos.DuraformOrder
{
    public class DuraformOrderForListDto
    {
        public Guid Id { get; set; }

        public int OrderNumber { get; set; }

        public string CustomerOrderNumber { get; set; }

        public OrderStatus OrderStatus { get; set; }

        public int DistributorId { get; set; }

        public int CabinetMakerId { get; set; }

        public string DistributorName { get; set; }

        public string CabinetMakerName { get; set; }

        public string Description { get; set; }

        public DateTime CreatedDate { get; set; }
    }
}